rpi5.yaml

```diff
@@ -0,0 +1,216 @@
+desc: "Raspberry 5 with xen dom0less"
+min_ver: "0.22"
+
+
+variables:
+  # Common build configuration
+
+  YOCTOS_WORK_DIR: "yocto"
+
+  DOM0_OS: "zephyr"
+
+  # Raspberry 5 build configuration
+
+  RPI_MACHINE: "raspberrypi5"
+  RPI_SOC_FAMILY: "bcm2712"
+
+  ZEPHYR_DOM0_DIR: "zephyr"
+  ZEPHYR_DOM0_MACHINE: "rpi_5_xen_dom0"
+  ZEPHYR_DOM0_TARGET: "zephyr/samples/hello_world"
+  ZEPHYR_DOM0_BUILD_DIR: "build-dom0"
+
+  DOMD_BUILD_DIR: "build-domd"
+  XT_XEN_DTB_NAME: "%{RPI_SOC_FAMILY}-%{RPI_MACHINE}-xen.dtb"
+  XT_DOMD_DTB_NAME: "%{RPI_SOC_FAMILY}-%{RPI_MACHINE}-domd.dtb"
+
+  XT_KERNEL_BRANCH: "rpi-6.6.y"
+  XT_KERNEL_REV: "${AUTOREV}"
+
+  XT_DOMD_IMAGE: "rpi5-image-minimal-domd"
+
+common_data:
+  # Sources used by domd
+  domd_sources: &DOMD_DOMU_SOURCES
```

Comment:
So, is this `domd_sources` or `DOMD_DOMU_SOURCES` ? Also, I can see DOMU anywhere

Reply:
Fixed